backend/api/placeapi.py

```diff
@@ -0,0 +1,83 @@
+import json
+
+import requests
+
+from .utils import get_api_key, get_secret_key
+
+
+class PlacesApiRequest:
+    """
+    Class for handling API requests from the Foursquare database.
+
+    Args:
+        longitude (float): The longitude of the desired location.
+        latitude (float): The latitude of the desired location.
+        radius (int): The maximum distance of a venue from the desired location.
+        limit (int): Limit of the results returned.
+        query (str): Query about the types of venues.
+
+    Attributes:
+        longitude (float): The longitude of the desired location.
+        latitude (float): The latitude of the desired location.
+        radius (int): The maximum distance of a venue from the desired location.
+        limit (int): Limit of the results returned.
+        query (str): Query about the types of venues.
+        response (list): Response given by Foursquare API.
+        venues (list): List of venues returned by API.
+    """
+
+    def __init__(self, longitude, latitude, radius, limit, query):
+        self.longitude = longitude
+        self.latitude = latitude
+        self.radius = radius
+        self.limit = limit
+        self.query = query
+        self.response = self.make_request()
+        self.venues = self.get_venues()
```

Comment:
This way we cannot instantiate an instance without making an external call.
Could we set `self.response` to `None` and use `make_request` when we actually want to do that :)?

Then the class name `PlacesApiRequest` could be changed to `PlacesApi` as we don't do any request on init.